service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/HealthcareServiceMapperTest.java

```diff
@@ -74,6 +79,7 @@ public void setup() {
         when(receivedOrganization.getNameArray(0)).thenReturn(name);
         when(name.getDomNode()).thenReturn(node);
         when(nodeUtil.getAllText(name.getDomNode())).thenReturn(HEALTHCARE_SERVICE_NAME);
+        lenient().when(informationRecipient.getTypeCode()).thenReturn(forString(PRCP_TYPE_CODE));
```

Comment:
Why lenient? In this case it seems unnecessary. I'd avoid lenient mocking in general - unless we have a good reason.